pipeline-modules/app-service/gcp/docker-compose-build-n-push.yaml

```diff
@@ -0,0 +1,118 @@
+provisioner: gcp
+name: docker-compose-build-n-push
+version: 1
+revision: 1
+displayName: Docker Compose Build & Push docker images
+description: This plugin only use to Run docker compose  build  & push command
+target: ""
+category: containerise
+keywords:
+  - Docker
+  - Build
+  - Push 
+  - Docker-compose
+    
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    docker_compose_service_name:
+      title: Service Name
+      description: "Service Name for which Docker image will build & push \n ,Note: Make sure Service name  must pe present in docker-compose.yaml"
+      type: string
+      default: service-name        
```

Comment:
this seems like it would be a required param and shouldn't have a default?

Reply:
Ya Service Name it is a required parameter , let me update this